src/components/footer/components/Logo.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Logo = () => (
+  <Link to="/" style={{ height: '36px' }}>
+    <svg
```

Comment:
You can simply save the svg from Zepling to a file like `logo.svg` or similar. [Then you import it like this](https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs). You probablydid some changes to the svg here, right? This gets annoying if the designer decides to change the file